component/web/oauth-common/src/main/java/org/gatein/security/oauth/openid/OpenIdProcessorImpl.java

```diff
@@ -426,34 +437,54 @@ protected JSONObject parseResponse(String httpResponse) throws JSONException {
 
         return uinfo;
     }
-    
+
     @Override
     public void revokeToken(OpenIdAccessTokenContext accessToken) throws OAuthException {
-    
+
     }
-    
+
     // Parse given String and add all scopes from it to given Set
     private void addScopesFromString(String scope, Set<String> scopes) {
         String[] scopes2 = scope.split(" ");
         for (String current : scopes2) {
             scopes.add(current);
         }
     }
-    
+
     protected URL sendAccessTokenRequest() throws IOException {
         if (log.isTraceEnabled())
             log.trace("AccessToken Request=" + this.accessTokenURL);
         return new URL(this.accessTokenURL);
     }
-    
+
     protected URL getTokenInfoURL() throws IOException {
         if (log.isTraceEnabled())
             log.trace("TokenInfo Request=" + this.tokenInfoURL);
         return new URL(this.tokenInfoURL);
     }
+
     protected URL getUserInfoURL() throws IOException {
         if (log.isTraceEnabled())
             log.trace("UserInfo Request=" + this.userInfoURL);
         return new URL(this.userInfoURL);
     }
+
+    @Override
+    public void start() {
+        if (StringUtils.isBlank(this.wellKnownConfigurationUrl)) {
+            log.warn("wellKnownConfigurationUrl is not configured");
+            return;
+        }
+        JsonObject json = this.remoteJwkSigningKeyResolver.getJson(this.wellKnownConfigurationUrl);
```

Comment:
Why do you use the remoteJwkSigningKeyResolver to read the configuration ?

The RemoteJwkSigningKeyResolver is used to get key to validate jwt token, not to read configuration